Base/usr/share/man/man1/tail.md

```diff
@@ -1,24 +1,39 @@
 ## Name
 
-tail
+tail - Print the end of a file
 
 ## Synopsis
 
-```sh
-$ tail [--follow] [--lines number] <file>
+```**sh
+$ tail [options]... [file]
 ```
 
 ## Description
 
-Print the end ('tail') of a file.
+`tail` prints the specified number (or 10) of lines at the end of `file` (or standard input).
```

Comment:
Same for (or standard input); make it clear that those are the defaults, the "or" is slightly confusing.